### ISSUE TITLE
Simulator adds 15 CP to real CP value.

### ISSUE THREAD

**Issue by leftbehind126** [doxxx/ffxiv-craft-opt-web#175]
Hopefully I am not doing something incorrectly. I've used the tool for a while now (thank you, btw) without this issue. Just recently saw it appear.

After setting the crafter attributes (in my case WVR) and starting the simulator, it will try to solve the recipe with an additional 15 CP. In the picture below, I have my attributes set to my base WVR stats without food.
![1_wvr](https://user-images.githubusercontent.com/1676991/29052509-9ee2bd8a-7bb8-11e7-97e9-6ce542c15435.jpg)

Once I start the simulator, it finishes with this:
![2_wvr](https://user-images.githubusercontent.com/1676991/29052525-b22997f6-7bb8-11e7-8f12-52d4e213ae92.jpg)

I also tried selecting food to see what that would do. It added 15 CP to that as well:
![3_wvr](https://user-images.githubusercontent.com/1676991/29052533-c87947cc-7bb8-11e7-85c1-66a2ce4a3152.jpg)

Please let me know if you have any questions and thank you for all of the hard work! It's appreciated!

**Comment by doxxx** [comment]
I've added an indicator in the status widget to show if the current class is marked as a specialist.

**Comment by Norggron** [comment]
There a way to turn this off? I don't have this trait and wont for a while, rendering any macro made useless.

**Comment by doxxx** [comment]
It shouldn’t be adding the cp unless you’ve checked the Speciailist box for your class. If you’re certai that you didn’t, please post the contents of the simulator log tab.

**Comment by Norggron** [comment]
I am a specialist, i haven't done the 70 quest, so i don't have the talent.

**Comment by doxxx** [comment]
Ah I see. The assumption was that if you're a specialist you have the Stroke of Genius trait as well. I'll see about adding another checkbox for Stroke of Genius.

**Comment by leftbehind126** [comment]
What I did was set my CP value to whatever my true CP value was minus 15. Since 15 CP gets added on the first step, the rest of the macro was not affected by it and it didn't cause any issues while generating a reliable macro. I just had to remember to do that any time I changed my gear. Ultimately, it just encouraged me to stop being lazy and to finish the quests in order to unlock the trait.

I am going to close this since it actually is **not** a bug.

doxxx - keep up the great work! This is an invaluable tool, and I sincerely appreciate all of the effort you've put into it. Thank you!
